.github/workflows/opt-out-export-unit-gf.yml

```diff
@@ -0,0 +1,44 @@
+name: opt-out-export-gf unit tests
+
+on:
+  pull_request:
+    paths:
+      - .github/workflows/opt-out-export-unit-gf.yml
+      - lambdas/attribution-data-file-share/**
+  workflow_dispatch:
+
+jobs:
+  test:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: codebuild-ab2d-${{github.run_id}}-${{github.run_attempt}}
+    env:
+      ACTIONS_ALLOW_USE_UNSECURE_NODE_VERSION: true
+    defaults:
+      run:
+        working-directory: ./lambdas/attribution-data-file-share
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
```

Comment:
@Sadibhatla these off-the-shelf actions are outdated, but they're sprinkled throughout the workflows in this project. While it's _probably_ fine here, there ought to be a separate effort encoded in Jira and scheduled for maintenance, upgrades.